okio/src/nonWasmTest/kotlin/okio/UseTest.kt

```diff
@@ -25,4 +25,11 @@ class UseTest {
 
     fakeFileSystem.checkNoOpenFiles()
   }
+
+  @Test
+  fun acceptsNullReturn() {
+    object : Closeable {
+      override fun close() {}
+    }.use { null }
```

Comment:
Can you add an assertNull or assertEquals here just for posterity.

Reply:
Resolved in commit f8dfad3